src/v2/Components/Artwork/Details.tsx

```diff
@@ -14,209 +19,217 @@ const TruncatedLine = styled.div`
+    v2: {
+      variant: "mediumText" as TextVariant,
+    },
+    v3: {
+      variant: "md" as TextVariant,
+    },
+  })
 
-    if (hideArtistName) {
-      return
-    }
+  if (cultural_maker) {
+    return (
+      <TruncatedLine>
+        <Text variant={tokens.variant}>{cultural_maker}</Text>
+      </TruncatedLine>
+    )
+  }
 
-    if (cultural_maker) {
-      return (
-        <TruncatedLine>
-          <Text variant="mediumText">{cultural_maker}</Text>
-        </TruncatedLine>
-      )
-    } else if (artists && artists.length) {
-      const artistLine = artists
-        .reduce((acc, artist, index) => {
-          return acc.concat([
-            ", ",
-            includeLinks
-              ? this.link(artist.name, artist.href, artist.id + "-" + index)
-              : artist.name,
-          ])
-        }, [])
-        .slice(1)
-      return (
-        <TruncatedLine>
-          <Text variant="mediumText">{artistLine}</Text>
-        </TruncatedLine>
-      )
-    }
+  if (artists && artists.length) {
+    return (
+      <TruncatedLine>
+        <Text variant={tokens.variant}>
+          {artists
+            .reduce((acc, artist, index) => {
+              return acc.concat([
+                ", ",
+                <ConditionalLink
+                  includeLinks={includeLinks}
+                  href={artist.href}
+                  key={index}
+                >
+                  <Text variant={tokens.variant}>{artist.name}</Text>
+                </ConditionalLink>,
+              ])
+            }, [])
+            .slice(1)}
+        </Text>
+      </TruncatedLine>
+    )
   }
 
-  titleLine() {
-    const { includeLinks } = this.props
-    const title = this.props?.artwork?.title
-    const date = this.props?.artwork?.date
-    const href = this.props?.artwork?.href
+  return null
+}
+
+const TitleLine: React.FC<DetailsProps> = ({
+  includeLinks,
+  artwork: { title, date, href },
+}) => {
+  const tokens = useThemeConfig({
+    v2: {
+      variant: "text" as TextVariant,
+    },
+    v3: {
+      variant: "md" as TextVariant,
+    },
+  })
 
-    const artworkText = (
+  return (
+    <ConditionalLink includeLinks={includeLinks} href={href}>
       <TruncatedLine>
-        <Text variant="text" color="black60">
-          {date ? title + ", " + date : title}
+        <Text variant={tokens.variant} color="black60">
+          {date ? `${title}, ${date}` : title}
         </Text>
       </TruncatedLine>
-    )
-    const artworkTextWithLink = <Link href={href}>{artworkText}</Link>
+    </ConditionalLink>
+  )
+}
 
-    return includeLinks ? artworkTextWithLink : artworkText
-  }
+const PartnerLine: React.FC<DetailsProps> = ({
+  includeLinks,
+  artwork: { collecting_institution, partner },
+}) => {
+  const tokens = useThemeConfig({
+    v2: {
+      variant: "text" as TextVariant,
+    },
+    v3: {
+      variant: "xs" as TextVariant,
+    },
+  })
 
-  line(text) {
+  if (collecting_institution) {
```

Comment:
I left all the snake_case aliasing alone because I wanted to not touch the tests or any other tests that relied on manual mocks.